lib/parse-message.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+module.exports = exports = {};
+
+
+exports.nickCommand = function(message, socket) {
+  console.log('username: ' + socket.username);
```

Comment:
Console.logs are great to have for your own testing purposes, but from now on, be more conscious of getting rid of them before pushing up code, just as a way to establish good habits of best practices (i.e. console.logs wouldn't be in deployed code). 

You can just have a branch with your console.logs and submit a branch that doesn't have the logs.